back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -0,0 +1,25 @@
+package com.ib.user;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "user_table") // Must not be "user".
```

Comment:
Why not just users, all other tables have names without _table?

Reply:
Fixed in [7d0115f](https://github.com/magley/sssscs/pull/1/commits/7d0115f5c68cdfaea1f8b9ba7165d68af75aa1dc).